lib/pbench/server/db/models/tracker.py

```diff
@@ -461,7 +429,7 @@ def create(**kwargs):
             return dataset
 
     @staticmethod
-    def attach(**kwargs):
+    def attach(path=None, controller=None, name=None, state=None):
```

Comment:
You didn't update the function docstring (e.g., lines 442-452) to match this change to the signature.

Reply:
Ugh. Documentation is such a pain. ;-)

But it turns out I have some stale text in here back from when it was `attach_or_create` that needed to be expunged anyway. (I can't, sadly, gloat that you missed it, since I did also...)

Reply:
Yeah, don't gloat -- you'll only spur me on, and I have the feeling that I'm finding too much as it is.  (I'm kind of embarrassed by what I turned in to Nikhil on the last round....)